pilot/pkg/features/pilot.go

```diff
@@ -418,6 +419,30 @@ var (
 		return strings.Split(trustedGatewayCIDR.Get(), ",")
 	}()
 
+	CATrustedNodeAccounts = func() map[types.NamespacedName]struct{} {
```

Comment:
How permanent is this intended to be? If we want to stop trusting some service account, we'd have to restart istiods.

Reply:
Intent is you have 1 ztunnel and that is the only one ever configured. We can move to mesh config if needed though

Reply:
The maybe impossible scenario I imagined is somehow something talking to istiod manages to hijack the identity of the ztunnel and grab all the node certs.